src/database/index.tsx

```diff
@@ -684,7 +740,7 @@ export const deleteFacility = async (
     //
     // * https://firebase.google.com/docs/firestore/manage-data/delete-data#web_2
     // ** https://github.com/Recidiviz/covid19-dashboard/issues/191
-    const facilityDocRef = await scenarioRef
+    const facilityDocRef = scenarioRef
```

Comment:
Do we no longer need to `await` or did we never need to `await` in the first place?

Reply:
Yeah we never needed it since `.doc()` doesn't return a promise, and removing it removed a linter warning.